drivers/SmartThings/zwave-thermostat/src/ct100-thermostat/init.lua

```diff
@@ -13,212 +13,73 @@
 -- limitations under the License.
 
 local capabilities = require "st.capabilities"
-local utils = require "st.utils"
---- @type st.zwave.CommandClass
 local cc = require "st.zwave.CommandClass"
 --- @type st.zwave.CommandClass.SensorMultilevel
-local SensorMultilevel = (require "st.zwave.CommandClass.SensorMultilevel")({ version = 5 })
---- @type st.zwave.CommandClass.ThermostatFanMode
-local ThermostatFanMode = (require "st.zwave.CommandClass.ThermostatFanMode")({ version = 3 })
---- @type st.zwave.CommandClass.ThermostatFanState
-local ThermostatFanState = (require "st.zwave.CommandClass.ThermostatFanState")({ version = 2 })
+local SensorMultilevel = (require "st.zwave.CommandClass.SensorMultilevel")({ version = 2 })
```

Comment:
<img width="564" alt="image" src="https://user-images.githubusercontent.com/20526748/214638298-456e2f20-9459-4304-bb8d-5b80a89ea3d6.png">
Looking at the spec, one of the added sensor types in v5 is humidity, which is likely why that version was used. Not sure if this will break anything, likely the device will handle everything just fine since msg formats didn't change between versions, but we should probably keep v5.

Reply:
the real frustrating thing is that, if they supported SMv5 why they put the sensors on two different endpoints instead of just using the sensor_type field in the get command

Reply:
Ahh okay, then Ill leave it to your discretion then, but I think this is one of those things where we would need to test with a real device.

Reply:
I'm asking community members to test, since I don't think we have access to one of these anymore.